wporg-gp-translation-events.php

```diff
@@ -135,13 +153,19 @@ function submit_event_ajax() {
 		}
 	}
 
-	$title        = sanitize_text_field( $_POST['event_title'] );
-	$description  = sanitize_text_field( $_POST['event_description'] );
-	$start_date   = sanitize_text_field( $_POST['event_start_date'] );
-	$end_date     = sanitize_text_field( $_POST['event_end_date'] );
-	$locale       = sanitize_text_field( $_POST['event_locale'] );
-	$project_name = sanitize_text_field( $_POST['event_project_name'] );
+	$title          = sanitize_text_field( $_POST['event_title'] );
+	$description    = sanitize_text_field( $_POST['event_description'] );
+	$start_date     = sanitize_text_field( $_POST['event_start_date'] );
+	$end_date       = sanitize_text_field( $_POST['event_end_date'] );
```

Comment:
Just a detail, we could rename `event_start_date` to `event_start` since it's not a date any more, but a datetime. Would also apply to `event_end_date`, and to the `meta_key` of each.